Libraries/Image/RCTImageLoader.mm

```diff
@@ -54,20 +53,37 @@ static NSInteger RCTImageBytesForImage(UIImage *image)
 #endif // [TODO(macOS GH#774)
 }
 
-#if TARGET_OS_OSX
+#if TARGET_OS_OSX // [TODO(macOS GH#774)
+
+/**
+ * Github #1611 - We can't depend on RCTUIKit here, because this file's podspec (React-RCTImage) doesn't
+ * take a dependency on RCTUIKit's pod `React-Core`. Let's just copy the methods we want to shim here
+ */
+
 static NSData *NSImageDataForFileType(NSImage *image, NSBitmapImageFileType fileType, NSDictionary<NSString *, id> *properties)
 {
   RCTAssert(image.representations.count == 1, @"Expected only a single representation since UIImage only supports one.");
 
   NSBitmapImageRep *imageRep = (NSBitmapImageRep *)image.representations.firstObject;
   if (![imageRep isKindOfClass:[NSBitmapImageRep class]]) {
-    RCTAssert([imageRep isKindOfClass:[NSBitmapImageRep class]], @"We need an NSBitmapImageRep to create an image.");
-    return nil;
+	RCTAssert([imageRep isKindOfClass:[NSBitmapImageRep class]], @"We need an NSBitmapImageRep to create an image.");
```

Comment:
Tabbing error?

Reply:
I guess we prefer spaces!
A future commit in RN Core does do a clang format so we should be able to avoid in the future!